src/views/product-downloads-view/index.tsx

```diff
@@ -83,21 +82,13 @@ const ProductDownloadsViewContent = ({
 			breadcrumbLinks={breadcrumbLinks}
 			sidecarSlot={<SidecarMarketingCard {...sidecarMarketingCard} />}
 		>
-			{/**
-			 * @TODO remove DevDotContent here. It's used for scroll-margin-top
-			 * on headings only. We should instead use g-offset-scroll-margin-top.
-			 * Asana:
-			 * https://app.asana.com/0/1202097197789424/1202370169937866/f
-			 */}
```

Comment:
🗒️ Making a note here as well for posterity: I've assigned the task that was in this TODO to you @zchsh! I also updated the columns of the task and added a link to this PR. 😊